src/create.cpp

```diff
@@ -562,10 +562,20 @@ namespace create {
     uint8_t cmd[2] = { OC_PLAY, songNumber };
     return serial->send(cmd, 2);
   }
+  
+    bool Create::isLeftWheel() const {
+    if (data->isValidPacketID(ID_BUMP_WHEELDROP)) {
+      return (GET_DATA(ID_BUMP_WHEELDROP) & 0x08) != 0;
+    }
+    else {
+      CERR("[create::Create] ", "Wheeldrop sensor not supported!");
+      return false;
+    }
+  }
 
-  bool Create::isWheeldrop() const {
```

Comment:
Similar to the cliff API, I think this should be kept in addition to the new function calls:

```c
return (isLeftWheeldrop() || isRightWheeldrop());
```